compiler/luci-interpreter/src/kernels/TestUtils.h

```diff
@@ -71,7 +71,6 @@ std::vector<::testing::Matcher<float>> ArrayFloatNear(const std::vector<float> &
 template <typename T>
 inline std::vector<T> quantize(const std::vector<float> &data, float scale, int32_t zero_point)
 {
-  assert(!std::is_floating_point<T>::value);
```

Comment:
I think we had better keep this assert. 

If you really want to remove this, you should change the below code `numeric_limits<T>::min ()` to `numeric_limits<T>::lowest()`.

Reply:
Okay, I got it! i will change `numeric_limits<T>::min()` to `numeric_limits<T>::lowest()`.

Reply:
If you don't use `quantizationParams` for testing with float values, this assert does not have to be removed.

Reply:
Okay, I will update it!